File: If-Else/Python/pr_03_largest_among_three_numbers.py
a = int(input())
b = int(input())
c = int(input())

if a > b:
    if a > c:
        print("The largest number is", a)
    else:
        print("The largest number is", c)

else:
    if b > c:
        print("The largest number is", b)
    else:
        print("The largest number is", c)

File: If-Else/CPP/pr_03_largest_among_three_numbers.cpp
#include<iostream>
using namespace std;

int main() {

    int a, b, c;
    cout<<"Enter the value of a ";
    cin>>a;
    cout<<"Enter the value of b ";
    cin>>b;
    cout<<"Enter the value of c ";
    cin>>c;

    if (a>b) {
        if (a>c) {
            cout<<a<<" is a larger number";
        }
        else {
            cout<<c<<" is a larger number";
        }
    }
    else {
        if (b>c) {
            cout<<b<<" is a larger number";
        }
        else {
            cout<<c<<" is a larger number";
        }
    }

    return 0;
}